redbeams/src/main/resources/schema/app/20210219153728_CB-10722_Set_AWS_root_cert_ID_when_creating_RDS.sql

```diff
@@ -0,0 +1,11 @@
+-- // CB-10722 Set AWS root cert ID when creating RDS
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE sslconfig
+    ADD COLUMN IF NOT EXISTS sslcertificateactivecloudprovideridentifier VARCHAR(255);
```

Comment:
do we need to update this field with a default value ?

Reply:
No, the code takes care of assuming a default value when needed. Currently, the only place where this happens is `DatabaseServerConfigToDatabaseServerV4ResponseConverter.convert()` that invokes `databaseServerSslCertificateConfig.getLegacyCloudProviderIdentifierByPlatform()` to fetch the default value. Please note that this is cloud platform dependent, just like the `sslcertificateactiveversion` column added earlier in #10068 which is also nullable and resolved using `databaseServerSslCertificateConfig.getLegacyMaxVersionByPlatform()`.

A similar default value logic will be also needed in a future PR that will take of cert rotation for the external DB.